.gitignore

```diff
@@ -8,3 +8,4 @@ target/
 /cli/pack/
 /.idea/
 /.idea_modules/
+bin/
```

Comment:
The new versions of sbteclipse always compile into `bin/` folders rather than `classDirectory.value`.